<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Honey" failures="1" tests="3" name="Test" time="8.223" errors="0" timestamp="2025-07-24T13:58:57 IST">
  <testcase classname="day46.OrangeHRM" name="testLogo" time="0.035"/>
  <testcase classname="day46.OrangeHRM" name="testUrl" time="0.014">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day46.OrangeHRM.testUrl(OrangeHRM.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testUrl -->
  <testcase classname="day46.OrangeHRM" name="testTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testTitle -->
</testsuite> <!-- Test -->
